datasets/acute-care/copy_slivar_image.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -ex
+
+gcloud auth configure-docker australia-southeast1-docker.pkg.dev
+skopeo copy docker://docker.io/brentp/slivar:latest docker://australia-southeast1-docker.pkg.dev/cpg-common/images/slivar
```

Comment:
Would it be worth pinning the version here?

Reply:
it would, 

I was wondering whether it would be worth making a generic form that takes the source and target locations as arguments, so copying in new containers wouldn't require a new script to be reviewed

On the flip side, I like the idea that new additions would be reviewed...

Reply:
@MattWellie, definitely +1 for this, I should have picked that up in the initial review.

Reply:
Ah you've already done it, easy :)